src/core/defaults.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Sets the core defaults
+ */
+export const name = "core/defaults";
+import { rule as checkInternalSlots } from "./linter-rules/check-internal-slots";
+import { rule as checkPunctuation } from "./linter-rules/check-punctuation";
+import linter from "./linter";
+import { rule as localRefsExist } from "./linter-rules/local-refs-exist";
+import { rule as noHeadinglessSectionsRule } from "./linter-rules/no-headingless-sections";
+import { rule as noHttpPropsRule } from "./linter-rules/no-http-props";
+
+linter.register(
+  noHttpPropsRule,
+  noHeadinglessSectionsRule,
+  checkPunctuation,
+  localRefsExist,
+  checkInternalSlots
+);
+
+const coreDefaults = {
```

Comment:
export here instead: 

```suggestion
export const coreDefaults = {
```